drivers/hmis/lib/form_data/tarrant_county/fragments/patches/sexual_orientation.json

```diff
@@ -6,6 +6,7 @@
         "type": "CHOICE",
         "link_id": "sexual_orientation",
         "text": "Sexual Orientation",
+        "warn_if_empty": false,
```

Comment:
This is not a HUD field so doesn't need a change. It is harmless since the field defaults to false, but recommend removing the change since it's not needed.

Reply:
This was part of the [issue](https://github.com/open-path/Green-River/issues/8234) "Remove "Warn if empty" behavior for Sexual Orientation field" - sounds like that was incorrect though. I can back it out